### ISSUE TITLE
[feature request] Shard dump utility that exports shards into line protocol format

### ISSUE THREAD

**Issue by rossmcdonald** [influxdata/influxdb#5531]
Similar to many of the other `dump` utilities out there for relational databases (`mysqldump` being a prime example). We've had this type of functionality in the past through the `shard-dump` branch's `inspect` utility, however it hasn't been tested with the latest release/storage engine and was never officially merged into master. Having a tool like this would greatly help simplify backups/restores, and would allow for more straightforward upgrade paths in the case of breaking changes.

Example use:

```
$ influx_inspect dump -path /var/lib/influxdb/data -database mydatabase -output /tmp/mydatabase.dump
```

Where you can specify the path to the data directories, the target database (or all of them), and then the target output path. Once complete, the output file would be a plain-text rendering of the shard data in pure line protocol format, which could then be re-imported using the `influx import` command.

**Comment by e-dard** [comment]
I think this is a good idea. 

This tool would also partly enable a simple streaming migration from one host to another with `influx_inspect dump ... | ssh user@other 'influx -import ...'`

**Comment by MikeSchroll** [comment]
How close does this get? https://github.com/influxdata/influxdb/pull/7046

**Comment by e-dard** [comment]
Closed via #7046
